generator/wrappers.py

```diff
@@ -121,6 +123,8 @@ def wrap_algo(algo, ver):
     'algorithms::em_gmm::init': ['seed',], # deprecated
     'algorithms::pca': ['covariance'], # parameter defined multiple times with different types
     'algorithms::kdtree_knn_classification': ['seed',], # deprecated
+    'algorithms::lasso_regression::training': ['optionalArgument', 'gramMatrix'], # internal stuff, optional input
+    'algorithms::lasso_regression::prediction': ['algorithms::linear_model::interceptFlag',], # parameter
```

Comment:
why is this ignored?

Reply:
optionalArgument is internal parameter for kernel calls which is not accessible for end users.
gramMatrix is optional input that is not ready for using now, ignoring is requested by DAAL team. It will be moved to optional args.
interceptFlag is common interface parameter which is not provided in prediction algo/stage directly, it is a part of model from training stage.